ember-toucan-core/rollup.config.mjs

```diff
@@ -24,7 +24,11 @@ export default {
     // These are the modules that should get reexported into the traditional
     // "app" tree. Things in here should also be in publicEntrypoints above, but
     // not everything in publicEntrypoints necessarily needs to go here.
-    addon.appReexports(['components/*.js', 'components/form/*.js']),
+    addon.appReexports([
+      'components/*.js',
+      'components/form/*.js',
+      'components/form/controls/*.js',
+    ]),
```

Comment:
Open to suggestions here - I played around with glob patterns but couldn't get _exactly_ what I wanted.  Essentially I wanted to ignore `components/form/-private` but include everything else, but everything I tried using the `!` pattern kept giving me not what I wanted.  This is a bit verbose 🤷 .  100% open to help/suggestions here!

Reply:
In headless-form I have `-private` as a direct child of `src`, so `-private/components/form` instead of `components/form/-private`, see https://github.com/CrowdStrike/ember-headless-form/tree/main/packages/ember-headless-form/src. It's a matter of taste I guess, so not strongly advocating for that file layout. But it would make the glob simple here, as `components/**/*.js` would cover only public components then.

Reply:
I like this idea! It also helps us be consistent between the two repos, which I like.  I went ahead and followed the same pattern as headless-form and will push the change in just a bit! Thanks for the suggestion!  I also like how the glob here is way simpler like you said with `components/**/*.js` only